inc/fields/class-field-post-select.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+class Shortcake_Field_Post_Select {
```

Comment:
This should technically be `Shortcode_UI`. All of our code-level references should be `shortcode-ui` for consistency.